src/lib/helpers/youtubeTranscriptsHandling.ts

```diff
@@ -1,7 +1,8 @@
 import { Innertube } from "youtubei.js";
+import type { CaptionTrackData } from "youtubei.js/PlayerCaptionsTracklist";
+import { HTTPException } from "hono/http-exception";
 
-// @ts-ignore to be fixed
-function createTemporalDuration(milliseconds) {
+function createTemporalDuration(milliseconds: number) {
     return new Temporal.Duration(
```

Comment:
I know the new Temporal.Duration isn't added by this PR, but Temporal.Duration.from({ milliseconds: number }); static method exists, which is much more simple to construct than new Temporal.Duration. I'd argue createTemporalDuration is a little redundant if Temporal.Duration.from is used instead.